tests/cypress/integration/actions_tasks_objects/case_44_changing_default_value_for_attribute.js

```diff
@@ -0,0 +1,78 @@
+// Copyright (C) 2021 Intel Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+import { taskName } from '../../support/const';
+
+context('Changing a default value for an attribute.', () => {
+    const caseId = '44';
+    const additionalLabel = `Case ${caseId}`;
+    const additionalAttrsLabel = [
+        { additionalAttrName: 'type', additionalValue: '', typeAttribute: 'Text' },
+        { additionalAttrName: 'shape', additionalValue: 'False', typeAttribute: 'Checkbox' },
```

Comment:
Not critical, but attribute names ``type`` and ``shape`` are actually confusing. Could you please use in the future more natural names (like ``color``, ``parked``, etc.)?